### DISCUSSION TITLE
Can you pass through DAX filters in Excel to power BI linked tables?

### DISCUSSION THREAD

**Discussion by NoahLull** [DaxStudio/DaxStudio#1114] [Q&A]
Is there a way to pass through filters to a Power BI model via a cell in excel to the excel properties command text box?

We have users that submit change request forms to us that are riddled with errors and want to build forms that are linked to the valid data.

Example would be where we pass through an Item Number and a facility number stored in cells in Excel and we have a canned DAX script that pulls back all the valid results and then excel would error check based off that.

**Comment by dgosbell** [comment]
You can do this with a macro in Excel. Marco showed a technique for doing this here https://www.sqlbi.com/tv/consuming-a-dax-query-in-excel-unplugged-5/
